python/katana/bug/environment.py

```diff
@@ -203,14 +205,16 @@ def is_interactive():
     return hasattr(sys, "ps1")
 
 
-def capture_environment(filename: Optional[Union[str, Path, Any]] = None):
+def capture_environment(filename: Optional[Union[str, Path, Any]] = None, **kwargs):
+    # pylint: disable=unused-argument
```

Comment:
To actually do anything with `kwargs` you need to pass it to the registered routines in `_environment_capture_routines`. I think you will want `f(zipout, **kwargs)`. Otherwise, the enterprise code will not be able to access the `client` arg to do anything useful with it.

Reply:
oh snap, yes you are correct. forgot the most important part. X.X